charter/charter.md

```diff
@@ -18,7 +18,7 @@ QUIC specifications:
 - Maintenance and evolution of the existing QUIC extensions specified
   by the WG.
 
-Maintenance and evolution work needs to be strongly motivated by
+WG adoption of maintenance and evolution work for the QUIC base specifications needs to be strongly motivated by
```

Comment:
Trying to capture @LPardue's suggestion:
```suggestion
WG adoption of work items falling into this first area of work needs to be strongly motivated by
```

Reply:
this is good, as the first area of work has base specifications and extensions (two bullets above this line).

Reply:
Works for me